src/olympia/blocklist/tests/test_commands.py

```diff
@@ -94,6 +111,18 @@ def test_single_guid(self):
         assert blocks[1].kinto_id == blocklist_json['data'][2]['id']
         assert blocks[1].include_in_legacy
         assert blocks[1].modified == datetime(2019, 11, 22, 16, 49, 58, 416000)
+        assert KintoImport.objects.count() == 6
+        assert KintoImport.objects.filter(
+            outcome=KintoImport.OUTCOME_NOMATCH).count() == 3
+        assert KintoImport.objects.filter(
```

Comment:
As mentioned above, this test isn't really about testing this, but rather about testing the single guid functionality. If this data could configured/altered by the test we could remove this assertion.